tests/admin_utils/tests.py

```diff
@@ -96,6 +95,13 @@ def test_relation_on_abstract(self):
 
 
 class UtilsTests(TestCase):
+
+    def setUp(self):
+        """
+        Set empty value to use with display_for_field() and display_for_value()
+        """
+        self.empty_value = '-empty-'
```

Comment:
is the value ever modified such that we need a setup method? if not , I think it would be better if it were simply a class attribute.